src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/emails/court/CourtEmailFactory.kt

```diff
@@ -14,7 +16,8 @@ import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.Prisoner
 import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.BookingAction
 import java.util.UUID
 
-object CourtEmailFactory {
+@Component
+class CourtEmailFactory(@Value("\${default.court.video.url:}") private val defaultCourtVideoUrl: String) {
```

Comment:
Too be honest I am little on the fence with whether having this as an env variable is worth it.

We could just have a method in here with it hard coded 🤷